typescript/test/electrum.test.ts

```diff
@@ -108,6 +108,28 @@ describe("Electrum", () => {
         })
       })
 
+      describe("getTransactionHistory", () => {
+        it("should return proper transaction history for the given address", async () => {
+          // https://live.blockcypher.com/btc-testnet/address/tb1qumuaw3exkxdhtut0u85latkqfz4ylgwstkdzsx
```

Comment:
Just confirming: this is an address controlled by us and it will not execute any new transactions in the future? 😆

Reply:
Yep, it is :)